src/plugins/gs-plugin-appstream.c

```diff
@@ -745,8 +745,6 @@ gs_plugin_add_categories (GsPlugin *plugin,
 		app = g_ptr_array_index (array, i);
 		if (as_app_get_id (app) == NULL)
 			continue;
-		if (as_app_get_priority (app) < 0)
```

Comment:
It'd be fixed, I guess. I'm not particularly sure how AppStream calculates the priority of the apps.